src/web/routes/application/email-address/email-address.test.js

```diff
@@ -1,6 +1,14 @@
 const test = require('tape')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
 const { contentSummary } = require('./email-address')
 
+const { TEXT, EMAIL } = require('../common/constants')
+
+const handleConfirmationCodeReset = sinon.spy()
+
+const { behaviourForPost } = proxyquire('./email-address', { '../common/confirmation-code': { handleConfirmationCodeReset } })
```

Comment:
I know PRs are not really about formatting so not required, but I would maybe reformat this to multiple lines as its quite a lot of inline `{ ... }` so a bit tricky to determine the nesting.